### PR TITLE
doc(xjl): added Chinese version of ppg doc

### PR DESCRIPTION
**Pull request by PommesPeter**
Please @zjowowen  review these changes

### CODE REVIEW

**Comment by PaParaZz1** [comment]
Thanks for you contribution. BTW, please modify the name of this pull request as our previous format.

File: source/12_policies/ppg_zh.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   :align: center
+   :height: 250
+
+重要公式
+-------------
+PPG 的优化分为两个阶段，策略阶段和辅助阶段。在策略阶段，策略网络和价值网络的更新方式类似于 PPO。在辅助阶段，使用联合损失将价值知识提取到策略网络中：
+
+.. math::
+
+    L^{j o i n t}=L^{a u x}+\beta_{c l o n e} \cdot \hat{\mathbb{E}}_{t}\left[K L\left[\pi_{\theta_{o l d}}\left(\cdot \mid s_{t}\right), \pi_{\theta}\left(\cdot \mid s_{t}\right)\right]\right]
```
Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:34]: \mathcal{L}^{\mathrm{joint}}=......
you can use latex in this way.

File: source/12_policies/ppg_zh.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+
+伪代码
+-----------
+
+on-policy 训练流程
+==============================
+以下流程图展示了 PPG 如何在策略阶段和辅助阶段之间进行交替
+
+.. image:: images/PPG.png
```
Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:50]: you need to commit the image.

File: source/12_policies/ppg_zh.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+4. PPG中有两个值网络。
+
+5. 在DI-engine的实现中，我们对离策略PPG使用了两个缓冲区，它们仅在数据使用上限（数据 "max_use" ）上有所不同。
+
+重要图示
+----------
+PPG 利用分开的策略和价值网络来减少目标之间的干扰。策略网络包括一个辅助价值头部，用于将价值的知识提取到策略网络中，具体的网络结构如下所示：
+
+.. image:: images/ppg_net.png
```
Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:24]: you need to commit the image.

File: source/12_policies/ppg_zh.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+   :align: center
+   :width: 600
+
+.. note::
+
+   在辅助阶段，PPG 还会对值网络进行额外的训练。
+
+off-policy 训练流程
+==============================
+DI-engine 还实现了具有两个具有不同数据使用约束（"max_use"）的缓冲区策略 PPG。其中，策略缓冲区提供策略阶段的数据，而值缓冲区提供辅助阶段的数据。整个训练过程类似于 off-policy PPO，但会以固定频率执行额外的辅助阶段。
```
Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:60]: “DI-engine 还实现了具有两个不同数据使用约束（"max_use"）的缓冲区的策略 PPG。”？

Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:60]: ”数据使用约束“->"数据使用次数约束"

File: source/12_policies/ppg_zh.rst [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+扩展
+-----------
+- PPG 可以与以下方法结合使用：
+
+  * GAE 或其他优势估计方法
+  * 多个缓冲区，具有不同的 "max_use" 约束
```
Comment (zjowowen) [inline source/12_policies/ppg_zh.rst:67]: 多个具有不同的最大数据使用次数限制的回访缓存。

File: source/12_policies/ppg_zh.rst (hunk 1 above)
Comment (PommesPeter) [reply source/12_policies/ppg_zh.rst:34]: okay, fixed

File: source/12_policies/ppg_zh.rst (hunk 4 above)
Comment (PommesPeter) [reply source/12_policies/ppg_zh.rst:60]: polished.

Comment (PommesPeter) [reply source/12_policies/ppg_zh.rst:60]: fixed

**Comment by PaParaZz1** [comment]
Thanks for your contribution, we will merge this PR now!

**Pull request merged**